lib/cron.js

```diff
@@ -181,14 +181,21 @@
 		/**
 		 * get next date that matches parsed cron time
 		 */
-		_getNextDateFrom: function(start) {
-			var date = moment(start);
+
+		_getNextDateFrom: function(start,zone) {
+			var date;
+			if(zone){
+				date=moment(start).tz(zone);
+			}
```

Comment:
The formatting here seems to be messed up. I'm going to add a eslint+prettier at some point in the near future. In the meantime can you change this to `} else {` and then being the last `}` in line? Thanks!

Reply:
So sorry about that :+1: